drools-wb-webapp/pom.xml

```diff
@@ -1246,26 +1261,199 @@
+                <source>../drools-wb-screens/drools-wb-globals-editor/drools-wb-globals-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-guided-dtree-editor/drools-wb-guided-dtree-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-guided-scorecard-editor/drools-wb-guided-scorecard-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/main/resources</source>
+
+                <source>../drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-client/src/main/java</source>
+                <source>../drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-client/src/main/resources</source>
+
+
+                <!--<source>${path.to.kie.wb.common}/kie-wb-common-widgets/kie-wb-common-ui/src/main/java</source>-->
+                <!--<source>${path.to.kie.wb.common}/kie-wb-common-widgets/kie-wb-common-ui/src/main/resources</source>-->
```

Comment:
Just clean up these commented bits. The files tend to get too long even with regular content.